datadog_checks_dev/datadog_checks/dev/tooling/commands/release/trello/testable.py

```diff
@@ -180,9 +181,21 @@ def pick_card_member(config: dict, author: str, team: str) -> Optional[str]:
 @click.option(
     '--update-rc-builds-cards', is_flag=True, help='Update cards in RC builds column with `target_ref` version'
 )
+@click.option(
+    '--move-cards',
+    is_flag=True,
+    help='Do not create a card for a change, but move the existing card from '
+    + '`HAVE BUGS - FIXME` or `HAVE BUGS - FIXME` to INBOX team',
```

Comment:
Should one of these `HAVE BUGS - FIXME` be `FIXED - Ready to Rebuild`?

Reply:
Good catch!

Reply:
@yzhan289 : Fixed.